README.md

```diff
@@ -35,6 +35,23 @@ To use this library, you need to copy `js/imgcache.js` into your project and imp
 ```html
 <script src="js/imgcache.js"></script>
 ```
+### npm
+
+```shell
+npm install imgcache.js
+```
+
+### bower
+
+```shell
+bower install imgcache.js --save
+```
+
+Add a `<script>` to your `index.html`:
+
+```html
+<script src="/bower_components/imgcache.js/js/imgcache.js"></script>
```

Comment:
*your mileage may vary* here for the path if you have set a different one in a .bowerrc file